### PR TITLE
test: community api

### PR DESCRIPTION
**Pull request by nguyennh4522**
**What does this PR do?**

-   [x] test: community api

### CODE REVIEW

File: pkg/handler/invites_test.go [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+	}
+
+	tests := []struct {
+		name             string
+		args             string
+		wantCode         int
+		wantResponsePath string
+	}{
+		{
+			name:             "200_ok",
```
Comment (trkhoi) [inline pkg/handler/invites_test.go:161]: can we add test for case 200 but list leaderboard empty ?

**Review by hoangharry — APPROVED** [review]

File: pkg/handler/invites_test.go [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+		log:      log,
+	}
+
+	tests := []struct {
+		name             string
+		query            string
+		wantCode         int
+		wantResponsePath string
+	}{
+		{
```
Comment (trkhoi) [inline pkg/handler/invites_test.go:41]: can add empty config invite ?

**Review by trkhoi — CHANGES_REQUESTED** [review]

File: pkg/handler/invites_test.go (hunk 2 above)
Comment (nguyennh4522) [reply pkg/handler/invites_test.go:41]: You mean test with guild does not have config? It's "404_guild_id_does_not_exist" testcase

File: pkg/handler/invites_test.go (hunk 1 above)
Comment (nguyennh4522) [reply pkg/handler/invites_test.go:161]: You mean the test with guild does not have leaderboard? It's "200_guild_does_not_have_leaderboard" testcase

**Review by trkhoi — APPROVED** [review]
